doc/attr.xml

```diff
@@ -1352,3 +1352,40 @@ gap> HamiltonianPath(g);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="VertexConnectivity">
+<ManSection>
+  <Attr Name="VertexConnectivity" Arg="digraph"/>
+  <Returns>An non-negative integer.</Returns>
+  <Description>
+    For a digraph <A>digraph</A> with set of vertices <C>V</C>, the attribute
+    <C>VertexConnectivity(<A>digraph</A>)</C> returns the least cardinality
+    <C>|S|</C> of a subset <C>S</C> of <C>V</C> such that the induced subdigraph
+    of <A>digraph</A> on <C>V \ S</C> is disconnected, or has at most one
+    vertex. <P/>
+
+    The algorithm makes <C>O(n - d - 1 + d * (d - 1) / 2)</C> calls to a
+    max-flow algorithm wich itself has complexity <C>O((n ^ 2) * e)</C>, where
+    <C>n</C> is the number of vertices of <A>digraph</A>, and <C>e, d</C> are
+    the number of edges and the minimum out-degree (respectively) of the
+    symmetric closure of the digraph that results by removing all multiple edges
+    of <A>digraph</A>.
```

Comment:
Wouldn't it be more straightforward to say "the number edges and the minimum degree of the underlying undirected graph of <A>digraph</A>."?